packs/technic/gt_new_horizons.json

```diff
@@ -0,0 +1,225 @@
+    "battlegear2",
+    "Baubles",
+    "bdlib",
+    "beebetteratbees",
+    "berriespp",
+    "BetterAchievements",
+    "betterbuilderswands",
+    "betterloadingscreen",
+    "betterquesting",
+    "BiblioCraft",
+    "BiblioWoodsBoP",
+    "BiblioWoodsForestry",
+    "BiblioWoodsNatura",
+    "BinnieCore",
+    "BiomesOPlenty",
+    "blocklimiter",
+    "BloodArsenal",
+    "bq_standard",
+    "BrandonsCore",
+    "bt",
+    "BuildCraft|Compat",
+    "BuildCraft|Core",
+    "BuildCraftOilTweak",
+    "CarpentersBlocks",
+    "catwalks",
+    "cb4bq",
+    "chisel",
+    "chiseltones",
+    "CodeChickenCore",
+    "CoFHCore",
+    "compactkineticgenerators",
+    "computronics",
+    "croploadcore",
+    "CustomMainMenu",
+    "defaultworldgenerator",
+    "detravscannermod",
+    "DraconicEvolution",
+    "dreamcraft",
+    "EMT",
+    "endercore",
+    "EnderIO",
+    "EnderStorage",
+    "EnderZoo",
+    "enhancedlootbags",
+    "enhancedportals",
+    "eternalsingularity",
+    "examplemod",
+    "extracells",
+    "ExtraUtilities",
+    "FastCraft",
+    "findit",
+    "FloodLights",
+    "ForbiddenMagic",
+    "Forestry",
+    "ForgeMultipart",
+    "ForgeRelocation",
+    "fw",
+    "gadomancy",
+    "galacticgreg",
+    "GalacticraftCore",
+    "GalacticraftMars",
+    "galaxyspace",
+    "gendustry",
+    "GraviSuite",
+    "gregtech_addon",
+    "gtneioreplugin",
+    "gtnhtcwands",
+    "gttweaker",
+    "HardcoreDarkness",
+    "HardcoreEnderExpansion",
+    "harvestcraft",
+    "harvestthenether",
+    "HelpFixer",
+    "hodgepodge",
+    "holoinventory",
+    "HungerOverhaul",
+    "IC2",
+    "ic2nei",
+    "IC2NuclearControl",
+    "ifu",
+    "IguanaTweaksTConstruct",
+    "InGameInfoXML",
+    "inventorytweaks",
+    "IronChest",
+    "ironchestminecarts",
+    "IronTank",
+    "itlt",
+    "JABBA",
+    "journeymap",
+    "kekztech",
+    "lootgames",
+    "LunatriusCore",
+    "MagicBees",
+    "malisiscore",
+    "malisisdoors",
+    "Mantle",
+    "MineTweaker3",
+    "miscutils",
+    "modtweaker2",
+    "MouseTweaks",
+    "MrTJPCoreMod",
+    "Natura",
+    "naturescompass",
+    "NEIAddons",
+    "neiintegration",
+    "NodalMechanics",
+    "NotEnoughItems",
+    "OpenBlocks",
+    "OpenComputers",
+    "openglasses",
+    "OpenMods",
+    "openmodularturrets",
+    "openprinter",
+    "opensecurity",
+    "overloadedarmorbar",
+    "p455w0rdlib",
+    "ProjectBlue",
+    "ProjRed|Compatibility",
+    "ProjRed|Core",
+    "ProjRed|Expansion",
+    "ProjRed|Exploration",
+    "ProjRed|Fabrication",
+    "ProjRed|Illumination",
+    "ProjRed|Transmission",
+    "questbook",
+    "Railcraft",
+    "RandomThings",
+    "reauth",
+    "RelocationFMP",
+    "ResourceLoader",
+    "RIO",
+    "Roguelike",
+    "RWG",
+    "SGCraft",
+    "sleepingbag",
+    "SpiceOfLife",
+    "spongemixins",
+    "StevesAddons",
+    "StevesCarts",
+    "StevesFactoryManager",
+    "StorageDrawers",
+    "StorageDrawersBop",
```

Comment:
`StorageDrawersBiomesOPlenty` :)

Reply:
I have picked the`modid` from the jar archives with an automated script.

Reply:
They must have updated/changed. As the one on the `mod.json` file is the full name. :)

Reply:
```sh
#!/usr/bin/env bash

# Populates a JSON array out of all modid from jar archive
# failed entries can be caused by different mcmod.info structure
# or malformed mcmod.info JSON,
# They will be listed as Failed Path to mod jar so they
# can be resolved manually.
#
find \
  ~/MultiMC/instances/GTNH/.minecraft/mods \
  -type f \
  -name '*.jar' \
  -exec bash -c '
set -o pipefail;
if ! {
  unzip -p "$1" mcmod.info |
    tr "\n" " " |
      jq -r ".[0].modid";
}; then
  echo failed "$1";
fi
' _ {} \; |
  jq -R . |
    jq -s '.'
```